example/README.md

```diff
@@ -0,0 +1,42 @@
+# Example Project for nexus_auth
+
+This is a simple example project demonstrating how to use the nexus_auth package with Django and Django Rest Framework.
+
+## Setup
+
+1. Create a virtual environment and activate it:
+
+```bash
+python -m venv .venv
+source .venv/bin/activate  # On Windows use: .venv\Scripts\activate
+```
+
+1. Install requirements:
+
+```bash
+pip install -r requirements.txt
+```
+
+1. Run migrations:
+
+```bash
+python manage.py migrate
+```
+
+1. Create a superuser:
+
+```bash
+python manage.py createsuperuser
+```
+
+1. Run the development server:
```

Comment:
before this, i think it would be a good idea to mention that they need to modify the nexus-auth settings in example_project/settings.py?